robot/web/stream/server.go

```diff
@@ -143,7 +143,7 @@ func (server *Server) AddStream(ctx context.Context, req *streampb.AddStreamRequ
 	defer span.End()
 	// Get the peer connection to the caller.
 	pc, ok := rpc.ContextPeerConnection(ctx)
-	server.logger.Infow("Adding video stream", "name", req.Name, "peerConn", pc)
+	server.logger.Infow("Adding video stream", "name", req.Name, "peerConn", fmt.Sprintf("%p", pc))
```

Comment:
Surprised `pc` doesn't already print the pointer address.

Reply:
Yeah, I would like that to be the default. We can make it the default.

Reply:
> We can make it the default

Definitely not high priority for me, but sounds good.